types/handler.d.ts

```diff
@@ -0,0 +1,7 @@
+import {Socket} from "net";
+
+export type strongErrorHandler = (
+  err: Error,
+  req: Express.Request,
+  res: Express.Response
+) => void | Socket;
```

Comment:
I believe the handler function is accepting four arguments including a callback, this is an Express requirement. Also the handler should not be returning `Socket`. If it does, then it's a bug in the implementation. Conceptually, the handler returns `void` and may invoke `next`.

```ts
export type strongErrorHandler = (
   err: Error,
   req: Express.Request,
   res: Express.Response,
   next: (err?: any) => void,
 ) => void;
```